### ISSUE TITLE
add auto-update for beppe

### ISSUE THREAD

**Issue by axelnilsson98** [bjornregnell/sigrid#35]
I think it would be a good idea for beppe to autoupdate in a way similar to sigrid-monitor. I think it would help in a room when the TA is idle to not have to update beppe so often

**Comment by bjornregnell** [comment]
The problem with that is that state updates migh get lost. So for bepe to be improved and reactive I think it must be rewritten to ScalaJS with dynamic DOM stuff.

Does it work for you to have both Sigrid Monitor (in a different window) and Beppe simultaneously?

**Comment by axelnilsson98** [comment]
right, I didn't think about that. Maybe having sigridmonitor aswell is a workaround that will have to do then
